packages/components/src/Input/InputSearch/_docs/InputSearch.stories.tsx

```diff
@@ -0,0 +1,54 @@
+import { Meta, StoryObj } from "@storybook/react"
+import { InputSearch } from "../index"
+
+const meta = {
+  title: "KAIO-staging/Inputs/InputSearch",
+  component: InputSearch,
+  args: {
+    id: "search",
+  },
+  parameters: {
+    a11y: {
+      config: {
+        rules: [
+          {
+            // Built with no label on purpose, to be used within `TextField` where label is present
+            id: "label",
+            enabled: false,
+          },
+        ],
+      },
+    },
+  },
+} satisfies Meta<typeof InputSearch>
+
+export default meta
+
+type Story = StoryObj<typeof meta>
+
+export const Playground: Story = {
+  parameters: {
+    docs: {
+      canvas: {
+        sourceState: "shown",
+      },
+    },
+  },
+}
+
```

Comment:
Did we want to give this local state so a user can type into this or save that for the `SearchField` component?